### DISCUSSION TITLE
After upgrade from 9.3.2 to 9.7.1 Home tabid object not set to an instance of an object content 0

### DISCUSSION THREAD

**Discussion by thabaum** [dnnsoftware/Dnn.Platform#4083] [General] [answered]
Default.aspx?tabid=13&error=Object+reference+not+set+to+an+instance+of+an+object.&content=0

I am trying to understand how after upgrading a site from version 9.3.2 to 9.7.1 I am getting the error in the URL and an error for the browser 

![image](https://user-images.githubusercontent.com/13577556/92962184-75783180-f425-11ea-9e4c-4299c919eb56.png)

Browsers insist there is a problem with redirection, one saying too many attempts

![image](https://user-images.githubusercontent.com/13577556/92966692-d820fb80-f42c-11ea-9b0e-f5ece6402e6f.png)

Sure seems to be something related to the redirect, so I am guessing possibly something in the web.config?  Does anything need to be looked into for changes needed when upgrading 9.3.2 => 9.7.1 ?

Or a Host/Portal setting that may cause an issue like this?

https://dnndocs.com/content/tutorials/troubleshooting/ts-too-many-redirects/index.html

I am going through steps as this looks similar to issues however I am not getting anywhere with two portals after installing update.

Thank you for any help here.

**Comment by valadas** [comment, marked as answer]
Did you look at the upgrade known issue here https://github.com/dnnsoftware/Dnn.Platform/releases/tag/v9.7.1 did you implement either the before upgrade or after upgrade documented workaround ?

**Comment by thabaum** [comment]
Thank you @valadas I missed those special notes in the release.
